src/main/java/racingcar/controller/RacingCarDto.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.controller;
+
+import racingcar.domain.RacingCar;
+
+public class RacingCarDto {
+
+    private final String name;
+    private final int position;
+
+    public RacingCarDto(String name, int position) {
+        this.name = name;
+        this.position = position;
+    }
+
+    public static RacingCarDto from(RacingCar racingCar) {
+        return new RacingCarDto(racingCar.getName(), racingCar.getPosition());
+    }
```

Comment:
```suggestion
    private RacingCarDto(String name, int position) {
        this.name = name;
        this.position = position;
    }

    public static RacingCarDto from(RacingCar racingCar) {
        return new RacingCarDto(racingCar.getName(), racingCar.getPosition());
    }
```
정적팩토리메소드로 객체 생성을 유도하시려면, 객체 생성자를 private으로 막는게 좋을 것 같네요 🙂